modules/olm-deleting-operators-from-a-cluster-using-web-console.adoc

```diff
@@ -19,19 +19,23 @@ endif::[]
 
 .Procedure
 
-. From the *Operators* → *Installed Operators* page, scroll or type a keyword into the *Filter by name* to find the Operator you want. Then, click on it.
+. From the *Operators* → *Installed Operators* page, scroll or type a keyword into the *Filter by name* field to find the Operator you want. Then, click on it.
 
-. On the right-hand side of the *Operator Details* page, select *Uninstall Operator* from the *Actions* drop-down menu.
+. On the right side of the *Operator Details* page, select *Uninstall Operator* from the *Actions* drop-down menu.
 +
-An *Uninstall Operator?* dialog box is displayed, alerting you that:
+An *Uninstall Operator?* dialog box opens, listing the Operands associated with the Operator.
 +
-[.small]
---
-*Operator VerticalPodAutoscaler will be removed from openshift-vertical-pod-autoscaler. Click on the checkbox below to also remove all Operands associated with this Operator. If your Operator configured off-cluster resources, these will continue to run and require manual cleanup.*
---
-+
-Any resources managed by the Operator, including CRDs and CRs, are not removed. The web console enables dashboards and navigation items for some Operators. To remove these after uninstalling the Operator, you might need to manually delete the Operator CRDs.
+. Choose one of the following options for the *Delete all operand instances for this Operator* checkbox:
+
+* To remove Operands managed by the Operator automatically, including custom resource definitions (CRDs) and custom resources (CRs), select the checkbox.
```

Comment:
Actually, not. Only the custom resources (CRs) will be removed. The custom resource definitions (CRDs) still exist.

Reply:
@jianzhangbjz Thank you. Changes made. 
PTAL

Reply:
Thanks! Anyway, I reported a bug for this CRD removing issue: https://bugzilla.redhat.com/show_bug.cgi?id=2007907 
waiting for the Devs confirmation, thanks!